docs/execution-providers/Cloud-ExecutionProvider.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cloud
+description: Instructions to infer an ONNX model remotely with a cloud endpoint
+parent: Execution Providers
+nav_order: 12
+redirect_from: /docs/reference/execution-providers/Cloud-ExecutionProvider
+---
+
+# Cloud Execution Provider
+
+The Cloud Execution Provider enables ONNX Runtime to invoke a cloud endpoint for inferenece, the endpoint must be deployed beforehand.
+Cloud Execution Provider is in preview stage, all API(s) and usage are subjuct to change.
+
+## Contents
+
+To consume the endpoint, ONNX Runtime must load the same model locally, customer could then use configured run options to choose to go remote.
```

Comment:
"same model" is misleading. We need to expand this by mentioning the use case being targeted in this preview.

Reply:
Corrected.